### PR TITLE
#31 Toward scalajs 1.0.0

### PR DESCRIPTION
**Pull request by mikla**
Support for Scala.js 1.0.0-RC2 

Blocked by https://github.com/scalacenter/scalajs-bundler/pull/319

### CODE REVIEW

**Comment by mikla** [comment]
Compatibility with 0.6. is broken so far.

File: core/js/src/main/scala/dtc/js/JSDate.scala [hunk 1]
```diff
@@ -119,3 +128,14 @@ object JSDate {
...
 }
+
+private[js] object JSDateCompatibility {
+
+  /**
+    * Since Scala.js 1, JSDate works with Doubles instead io Ints.
+    * See https://github.com/scala-js/scala-js/issues/2751.
+    * This is just to reduce amount of back and forth conversions between Int and Double.
+    */
+  implicit def doubleToInt(x: Double): Int = x.toInt
```
Comment (vpavkin) [inline core/js/src/main/scala/dtc/js/JSDate.scala]: kinda ugly, but WYGD

Comment (mikla) [reply core/js/src/main/scala/dtc/js/JSDate.scala]: Yeah, otherwise you have to have 2 versions if JSDate to 0.6 and for 1.0. Which is also... meh..

Comment (vpavkin) [reply core/js/src/main/scala/dtc/js/JSDate.scala]: fair

**Comment by mikla** [comment]
@vpavkin I had to disable scoverage (runtime) as it's not ready for Scala.js 1.0.0.

**Pull request merged**